shell/common/shell.cc

```diff
@@ -1371,15 +1371,26 @@ void Shell::LoadDartDeferredLibrary(
 void Shell::LoadDartDeferredLibraryError(intptr_t loading_unit_id,
                                          const std::string error_message,
                                          bool transient) {
-  engine_->LoadDartDeferredLibraryError(loading_unit_id, error_message,
-                                        transient);
+  task_runners_.GetUITaskRunner()->PostTask(
```

Comment:
Consider using `  fml::TaskRunner::RunNowOrPostTask(` here and below.

Reply:
done